src/ape_plugins/_cli.py

```diff
@@ -223,3 +199,55 @@ def uninstall(cli_ctx, plugins, skip_confirmation):
 
     if failures_occurred:
         sys.exit(1)
+
+
+@cli.command()
+def update():
+    """
+    Update Ape and all plugins to the next version
+    """
+
+    _change_version(ape_version.next_version_range)
+
+
+def _version_callack(ctx, param, value):
+    obj = Version(value)
+    version_str = f"0.{obj.minor}.0" if obj.major == 0 else f"{obj.major}.0.0"
+    return f"=={version_str}"
+
+
+@cli.command()
+@click.argument("version", callback=_version_callack)
+def change_version(version):
+    """
+    Change ape and all plugins version
+    """
+
+    _change_version(version)
+
+
+def _change_version(spec: str):
+    # Update all the plugins.
+    # This will also update core Ape.
+    # NOTE: It is possible plugins may depend on each other and may update in
+    #   an order causing some error codes to pop-up, so we ignore those for now.
+    for plugin in _pip_freeze.get_plugins():
+        logger.info(f"Updating {plugin} ...")
+        name = plugin.split("=")[0].strip()
+        subprocess.call([sys.executable, "-m", "pip", "install", f"{name}{spec}", "--quiet"])
+
+    # This check is for verifying the update and shouldn't actually do anything.
+    logger.info("Updating Ape core ...")
+    completed_process = subprocess.run(
+        [sys.executable, "-m", "pip", "install", f"eth-ape{spec}", "--quiet"]
+    )
```

Comment:
question: Would this still not produce the same issue?  If you're running this it will install packaged ape rather than local editable, wouldn't it?

Reply:
I am attempting to address the 3rd bullet here in the issue link with the new command, it is not related to the primary issue but supposed to be a way to upgrade the full suite easily... What issue do you mean regarding the editable install?

Reply:
The issue that made me create #1890 that skips the plugin installation integration tests.  If that's unrelated, then ignore.